packer/ansible/agent.yml

```diff
@@ -11,8 +11,31 @@
           - java-1.8.0-openjdk.x86_64
           - git
           - jq
+          - gettext
+          - unzip
+          - python3
+          - mariadb # to be honest we need only client but centos doesn't have client package
+          - https://rpm.nodesource.com/pub_14.x/el/7/x86_64/nodejs-14.18.1-1nodesource.x86_64.rpm
         state: present
 
+    - name: Install docker-compose
+      get_url:
+        url: https://github.com/docker/compose/releases/download/v2.1.1/docker-compose-linux-x86_64
+        dest: /usr/local/bin/docker-compose
+        mode: 'u+x,g+x,o+x'
+
+    - name: Download and unarchive installer
+      unarchive:
+        src: https://s3.amazonaws.com/aws-cli/awscli-bundle.zip
+        dest: /tmp
+        remote_src: yes
+
+    - name: Install awscl
+      shell: python3 /tmp/awscli-bundle/install
```

Comment:
why do we install awscli?

Reply:
@BupycHuk it's the packer file for the Jenkins agent, not for PMM. We use aws-cli when we build PMM (for uploading files to S3 for example).